roles/ceph-docker-common/tasks/pre_requisites/prerequisite_docker.yml

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
This file is in the `ceph-docker-common` role already, do we really need to call this file with `_docker.yml`? :) Maybe we can just call it `prerequisites.yml`?